src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -26,6 +26,12 @@
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
 
+        <!-- Json Normalizer -->
+        <service id="serializer.normalizer.json" class="Symfony\Component\Serializer\Normalizer\JsonSerializableNormalizer" public="false">
```

Comment:
I guess the obvious answer is to add the service in the `FrameworkBundle::registerSerializerConfiguration` method after a check to see if the class exists, something similar seems to happen with some other parts of the configuration.
Alternatively there doesnt seem to be an issue with just adding the `ObjectNormalizer` to the config when it was added